Resources/Storage/SRM2Storage.py

```diff
@@ -82,7 +82,7 @@ def __init__( self, storageName, protocol, path, host, port, spaceToken, wspath
     self.gfalRetry = gConfig.getValue( "/Resources/StorageElements/GFAL_Retry", 3 )
 
     # # set checksum type, by default this is 0 (GFAL_CKSM_NONE)
-    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", 0 )
+    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", None )
```

Comment:
I agree. In this call the returned value is casted to the type of the default value. In case of None this will be a string returned if any value defined in the CS, None otherwise. So, if you understand what you are doing, then it is OK.